guides/common/modules/proc_upgrading-project-in-place-using-leapp.adoc

```diff
@@ -1,27 +1,19 @@
 [id="upgrading-project-in-place-using-leapp_{context}"]
-ifdef::foreman-el,katello[]
-= Upgrading {Project} to Enterprise Linux 8 In-Place Using Leapp
-endif::[]
-ifdef::satellite[]
-= Upgrading {Project} to {RHEL} 8 In-Place Using Leapp
-endif::[]
+ifdef::satellite,foreman-el,katello[]
```

Comment:
IMHO this should be moved to the assembly that includes this file.

Reply:
I think you're right and in fact, it's already there. That makes the ifdefs redundant.